anonymizer/base.py

```diff
@@ -337,12 +340,10 @@ def create_replacer_attributes(self):
         return tuple(name for name, replacer in self.get_attributes() if replacer != 'SKIP')
 
     def create_query(self, replacer_attrs):
-        query = 'UPDATE %s SET ' % self.model._meta.db_table
-        for attr in replacer_attrs:
-            query += '%s = ?, ' % attr
-        query = query[:-2] + ' where %s = ?' % self.model._meta.pk.column
-        # django expects %s, not ?, but we use %s to generate the query
-        return query.replace('?', '%s')
+        return 'UPDATE %s SET %s WHERE %s = %%s' % (
```

Comment:
This is a speed improvement because building a string in a for loop is quadratic. For fun, see https://accidentallyquadratic.tumblr.com/post/142387131042/nodejs-left-pad